samples/core/pom.xml

```diff
@@ -238,6 +239,11 @@
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-ws</artifactId>
+      <exclusions>
+      <exclusion>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-web</artifactId>
+        </exclusion></exclusions>
```

Comment:
save actions seem not to apply here. Please always format your code before comitting.